app/web/features/profile/locales/en.json

```
@@ -105,9 +105,9 @@
     "yes": "Yes"
   },
   "sleeping_arrangement": {
-    "private": "Private space",
-    "common": "Common area",
-    "shared_room": "Shared room"
+    "private": "Private space (e.g. own room or similar)",
```

I think "private_ space" is a bit too ambiguous. We should be explicit here to avoid any confusion:

```suggestion
    "private": "Private room (a room to yourself)",
```